### PR TITLE
Add anonymized Jira Inquiries

### PR DESCRIPTION
**Pull request by imkacarlson**
This includes 2 new pages where we can publish anonymized 170.315(g)(10) and 170.404 inquiries in their own respective pages.

Links to these new pages have been added to the (g)(10) and 404 home pages:

**Link on (g)(10) homepage**:
image

**Link on 170.404 homepage**:
image

Then those new links take you to a page with the anonymized inquiries:
image

### CODE REVIEW

File: docs/inquiry-portal/g10-inquiries.md [hunk 1]
```diff
@@ -0,0 +1,271 @@
...
+
+### February 2022
+**Stakeholder Inquiry**: We are seeking some further confirmation in relation to the [1] new clarification published to the (g)(10) Certification Companion Guide (CCG) on the use of capability URLs. Can our Health IT Module require clients to re-direct to a capability URL with an authorization header?
+
+[1] Health IT Modules may use access control schemes other than OAuth 2.0 for controlling access to the file server, such as capability URLs. The HL7 FHIR-I Work Group has documented expectations for the use of capability URLs with the Bulk Data Access IG on the HL7 confluence website. For purposes of Certification testing, Health IT Modules will be tested for the ability to share bulk data files either using OAuth 2.0 bearer tokens or via capability URLs accessible without preconditions or additional steps.
```
Comment (scottbohon) [inline docs/inquiry-portal/g10-inquiries.md:106]: Consider having the clarification in italics.

File: docs/inquiry-portal/g10-inquiries.md [hunk 2]
```diff
@@ -0,0 +1,271 @@
...
+- A Health IT Module’s authorization server must issue a refresh token for a period of no less than three months to native applications capable of securing a refresh token.
+
+and for subsequent connections:
+
+- Access must be granted to patient data in accordance with the implementation specification adopted in the HL7® SMART Application Launch Framework Implementation Guide (incorporated by reference at § 170.215(a)(3)) without requiring re-authorization and re-authentication when a valid refresh token is supplied by the application.
+- A Health IT Module’s authorization server must issue a refresh token valid for a new period of no less than three months to applications capable of storing a client secret. 
+Beyond the certification criteria requirements, health IT developers are not required to support all methods that third-party application developers seek to use. Additional information and clarifications regarding this criterion can be found on the <a target = "_blank" href = "https://www.healthit.gov/test-method/standardized-api-patient-and-population-services#ccg">September</a>.
```
Comment (scottbohon) [inline docs/inquiry-portal/g10-inquiries.md:189]: Review the sentence "Additional information and clarifications regarding this criterion can be found on the <a target = "_blank" href = "https://www.healthit.gov/test-method/standardized-api-patient-and-population-services#ccg">September</a>." "September" should likely instead be "(g)(10) CCG".

File: docs/inquiry-portal/g10-inquiries.md [hunk 3]
```diff
@@ -0,0 +1,271 @@
...
+
+## Paragraph (g)(10)(v)(B): Authentication / Authorization for Multiple Patient Services
+### October 2021
+**Stakeholder Inquiry**: There is a requirement that clients downloading the bulk data use the same access token as they did to make the original request. Does it permit other equivalently secure means of authorization as well?
+
+The  FHIR specification says that there should be alternatives: https://hl7.org/fhir/uv/bulkdata/export/index.html#response---complete-status
```
Comment (scottbohon) [inline docs/inquiry-portal/g10-inquiries.md:195]: Consider making a clickable hyperlink.

File: docs/inquiry-portal/g10-inquiries.md [hunk 4]
```diff
@@ -0,0 +1,271 @@
...
+
+In the provided example, the health IT developer could use "relied upon software" in the form of an authorization server to fulfill authentication and authorization requirements in the § 170.315(g)(10) "Standardized API for patient and population services" criterion. However, a Health IT Module cannot be certified to the § 170.315(g)(10) "Standardized API for patient and population services" criterion without fulfilling all of its specified requirements, including authentication and authorization requirements.
+For more information about "relied upon software", please see the <a target = "_blank" href = "https://www.healthit.gov/sites/default/files/relieduponsoftwareguidance.pdf">Relied Upon Software Program Guidance document</a>.
+
+### December 2021
+**Stakeholder Inquiry**: If we certify to 170.315(g)(10), does that cover all of the API requirements specified in this PDF? https://www.healthit.gov/cures/sites/default/files/cures/2020-03/APICertificationCriterion.pdf
```
Comment (scottbohon) [inline docs/inquiry-portal/g10-inquiries.md:26]: Consider making a clickable hyperlink.

File: docs/inquiry-portal/g10-inquiries.md [hunk 5]
```diff
@@ -0,0 +1,271 @@
...
+
+### August 2021
+**Stakeholder Inquiry**: US Core 3.1.1 requires that a server demonstrate support for the "composite OR" search for the CareTeam.status search parameter (US Core 3.1.1 CareTeam Mandatory Search Parameters). However, neither US Core, nor USCDI require that multiple CareTeam resource be supported. Can you clarify what is required here?
+
+**ONC Response**: According to the regulation text for the 170.315(g)(10) certification criterion at 45 CFR 170.315(g)(10)(i )(A), health IT developers must demonstrate the ability of Health IT Modules to:
```
Comment (scottbohon) [inline docs/inquiry-portal/g10-inquiries.md:75]: Consider replacing "(i )" with "(i)", removing the space.

File: docs/inquiry-portal/g10-inquiries.md [hunk 6]
```diff
@@ -0,0 +1,271 @@
...
+
+***
+**Stakeholder Inquiry**: Our FHIR API returns additional resource types in search responses and Inferno ((g)(10) Standardized API Test Kit) is failing us for this. Is this intended to not be allowed? The FHIR specification provides flexibility for server search responses.
+
+**ONC Response**: The Inferno <a target = "_blank" href = "https://github.com/onc-healthit/onc-certification-g10-test-kit/releases/tag/v2.2.2">ONC Certification (g)(10) Standardized API Test Kit v2.2.2</a>, released on 7/11/2022, includes the following update which should resolve the technical concern raised. If your implementation continues to experience this issue, please submit another ticket through the Health IT Feedback and Inquiry Portal.
+
+*Relaxes constraint that prevented systems from returning additional resource types in search responses. Systems may return other resource types if they believe them to be relevant, and tests now provide informational messages in this case.*
+
+### April 2022
+**Stakeholder Inquiry**: Does ONC have a published list of IP addresses we can whitelist for the (g)(10) Standardized API Test Kit hosted at https://inferno.healthit.gov/onc-certification-g10-test-kit?
```
Comment (scottbohon) [inline docs/inquiry-portal/g10-inquiries.md:235]: Consider making a clickable hyperlink.

File: docs/inquiry-portal/404-inquiries.md [hunk 7]
```diff
@@ -0,0 +1,80 @@
...
+
+We are not able to provide individualized advice on whether a specific fact pattern would or would not satisfy an exception or constitute information blocking.
+
+Anyone who believes they may have experienced or observed information blocking by <a target = "_blank" href = "https://www.healthit.gov/cures/sites/default/files/cures/2020-03/InformationBlockingActors.pdf">any health care provider, health IT developer of Certified Health IT, or health information network or health information exchange</a> is encouraged to share their concerns with us through the <a target = "_blank" href = "https://healthit.gov/report-info-blocking">Information Blocking Portal</a> on ONC’s website, HealthIT.gov.
+
+## Paragraph (b)(3): Rollout of (g)(10)-Certified APIs
+### May 2022
+**Stakeholder Inquiry**: According to the 21st Century Cures Act: Interoperability, Information Blocking, and the ONC Health IT Certification Program Final Rule (ONC Cures Act Final Rule) timeline, ONC Cures Update Certification Criteria must be made available by 12/31/2022. Can you clarify what made available means?
```
Comment (scottbohon) [inline docs/inquiry-portal/404-inquiries.md:31]: Consider putting "made available" in quotes in the last sentence.

File: docs/inquiry-portal/404-inquiries.md [hunk 8]
```diff
@@ -0,0 +1,80 @@
...
...the ONC Health IT Certification Program, it must meet this Maintenance of Certification requirement no later than December 31, 2022, to continue participating in the ONC Health IT Certification Program. Failure to comply with a Condition and Maintenance of Certification requirement could result in a Certified Health IT Module being non-compliant, and ONC could initiate Direct Review (<a target = "_blank" href = "https://www.ecfr.gov/current/title-45/subtitle-A/subchapter-D/part-170/subpart-E/section-170.580#p-170.580(a)(2)(iii)">45 CFR 170.580(a)(2)(iii)</a>) and pursue corrective actions to enforce the requirement. ONC’s goal is to work with Certified Health IT Developers to remedy any non-conformities in a timely manner, but failure to conform with requirements of the ONC Health IT Certification Program can ultimately result in terminating the affected Health IT Modules and/or issuing a certification ban to the Certified Health IT Developer. For more information on Direct Review, please visit http://www.healthit.gov/Direct-Review and our <a target = "_blank" href = "https://www.healthit.gov/sites/default/files/page/2022-03/Draft_Review_Factsheet.pdf">Direct Review fact sheet</a>.
```
Comment (scottbohon) [inline docs/inquiry-portal/404-inquiries.md:67]: Consider a clickable link for "http://www.healthit.gov/Direct-Review"

**Review by scottbohon — CHANGES_REQUESTED** [review]
I added comments for your consideration.

**Comment by imkacarlson** [comment]
Thanks Scott!

Made updates based off your suggestions.

Also added links to the corresponding inquiry sections on the Information and Clarifications pages:

image

File: docs/inquiry-portal/g10-inquiries.md [hunk 9]
```diff
@@ -0,0 +1,271 @@
...
+
+The ONC Certification criteria for eCQMs does not require any FHIR standards. For additional information on CMS regulations, please reference CMS sources or contact CMS if you have questions about specific requirements. The CMS.gov website is a great place to start for information about the requirements of any particular CMS program that may be of interest to health care providers and those who support them with health IT solutions. For one example, the Quality Payment Program overview page (https://qpp.cms.gov/about/qpp-overview) offers links to educational resources and information how to contact CMS. Similarly, the CMS.gov Promoting Interoperability Programs page (https://www.cms.gov/Regulations-and-Guidance/Legislation/EHRIncentivePrograms) currently indicates that Medicare and dually eligible hospitals participating in the Medicare and Medicaid Promoting Interoperability Programs may contact the QualityNet help desk for assistance.
```
Comment (scottbohon) [inline docs/inquiry-portal/g10-inquiries.md:56]: Consider clickable links for links in this paragraph.

**Review by scottbohon — COMMENTED** [review]
Made a comment for your consideration.

**Pull request merged**